coralproject/talk-fe
# Embed code bugfix

## What does this PR do?

Addresses bugs introduced by language changes in the last PR.

## How do I test this PR?
From the talk-fe directory: cd packages/coral-embed-stream 2.npm run build` (Note, we should discuss how to automate this process as part of a broader build script, but this seems like a larger conversation.)
cd ../coral-admin
npm start
In a web browser, navigate to http://localhost:3142/embedlink/
Copy the link into a simple webpage.
Webpage should include an embedded comment stream.
